src/main/webapp/ts/profileBuilder.ts

```diff
@@ -14,11 +14,38 @@ function setForm(value: string) {
             menteeForm.setAttribute('style','display:block;');
             mentorForm.setAttribute('style','display:none;');
             userTypeButton.setAttribute('value','Mentee');
+            const menteeName = document.getElementById('name-input') as HTMLInputElement;
+            const menteeSchool = document.getElementById('school-input') as HTMLInputElement;
+            const menteeMajor = document.getElementById('major-input') as HTMLInputElement;
+            if(menteeName && menteeSchool && menteeMajor) {
+                menteeName.required = true;
+                menteeSchool.required = true;
+                menteeMajor.required = true;
+            }
+            else{
+                console.error("Required forms failed for mentee form");
+            }
         }
         else {
             mentorForm.setAttribute('style','display:block;');
             menteeForm.setAttribute('style','display:none;');
             userTypeButton.setAttribute('value','Mentor');
+
+            const mentorName = document.getElementById('profName-input') as HTMLInputElement;
```

Comment:
Can you think of any ways to minimize duplicated logic for the form validation?

Reply:
I'll see. The logic is duplicated bc the form Id's are different. Making them the same would make more lines of code to be changed in other files.

Reply:
Done